documentation/configuring_bolt.md

```diff
@@ -107,6 +103,23 @@ directory](projects.md).
 You can view a full list of the available options in [`bolt.yaml`
 options](bolt_configuration_reference.md).
 
+#### Updating project-level `bolt.yaml`
+
+You can update to use the latest Bolt best practices using our migration command:
+
+_\*nix shell command_
+
+```shell
+bolt project migrate
+```
+
+_PowerShell cmdlet_
+
+```powershell
+Update-BoltProject
+```
```

Comment:
We might want to just create a doc for upgrading to Bolt 3.0 that includes this information, similar to the doc we had for [migrating to Bolt 2.0](https://puppet.com/docs/bolt/latest/migrating_inventory_files.html).

Reply:
Ok, the doc I added is definitely not comprehensive but I figure we'll build it up as we get closer to 3.0